Tests/Tests/STPPaymentMethodCardChecksTest.m

```diff
@@ -0,0 +1,34 @@
+//
+//  STPPaymentMethodCardChecksTest.m
+//  StripeiOS Tests
+//
+//  Created by Yuki Tokuhiro on 3/5/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+@import XCTest;
+
+#import "STPPaymentMethodCardChecks.h"
+
+@interface STPPaymentMethodCardChecksTest : XCTestCase
+
+@end
+
+@implementation STPPaymentMethodCardChecksTest
+
+- (void)testDecodedObjectFromAPIResponse {
+    NSDictionary *response = @{@"address_line1_check": [NSNull null],
+                               @"address_postal_code_check": [NSNull null],
+                               @"cvc_check": [NSNull null]};
```

Comment:
It's probably worth updating/adding to this test so that it's checking the possible values (pass, failed, unavailable, unchecked), possibly along with the work to add an enum for them.